apps/extension/src/utils/mcpClient.ts

```
@@ -0,0 +1,381 @@
+			});
+
+			console.log('🔍 DEBUGGING: Transport created, checking process...');
+
+			// Set up transport event handlers
+			this.transport.onerror = (error: Error) => {
+				console.error('❌ MCP transport error:', error);
+				console.error('Transport error details:', {
+					message: error.message,
+					stack: error.stack,
+					code: (error as any).code,
+					errno: (error as any).errno,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)